src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public class ColorSupplier {
+
+    public String getRandomColor() {
+        String[] color = {"white", "black", "green", "red", "blue", "pink",
+                "yellow", "orange", "lime", "brown"};
+        return color[(int) (Math.random() * 10)];
```

Comment:
```suggestion
        return color[(int) (Math.random() * color.length)];
```

it's better to avoid using "magic" numbers